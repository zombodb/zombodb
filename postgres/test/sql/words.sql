SELECT count(*) FROM words;

SELECT count(*) FROM (SELECT words.word, assert(count(*), 1, words.word) FROM words INNER JOIN (SELECT word FROM words ORDER BY random() limit 2500) list ON words.word = list.word WHERE zdb(words) ==> ('word:"' || list.word || '"')::text GROUP BY words.word) x;

SELECT zdb_estimate_count('words', '') = (SELECT count(*) FROM words);
