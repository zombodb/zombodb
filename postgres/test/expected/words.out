SELECT count(*) FROM words;
 count  
--------
 310546
(1 row)

SELECT count(*) FROM (SELECT words.word, assert(words.word, 1, count(*)::int) FROM words INNER JOIN (SELECT word FROM words ORDER BY random() limit 2500) list ON words.word = list.word WHERE zdb(words) ==> ('word:"' || list.word || '"')::text GROUP BY words.word) x;
 count 
-------
  2500
(1 row)

SELECT zdb_estimate_count('words', '') = (SELECT count(*) FROM words);
 ?column? 
----------
 t
(1 row)

